src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHandlerManager.cs

```diff
@@ -91,6 +99,7 @@ private void HandleDesignTime(IProjectVersionedValue<IProjectSubscriptionUpdate>
         {
             Assumes.False(update.Value.ProjectChanges.Count == 0, "CPS should never send us an empty design-time build data.");
 
```

Comment:
We kick off design-time builds for all "buildable" configs in a multi-targeted project.